addons/overheating/XEH_postInit.sqf

```diff
@@ -76,8 +76,18 @@ if (hasInterface) then {
         }] call CBA_fnc_addClassEventHandler;
     };
 
+    // Reset all weapon heat to ambient on death to prevent cookoffs when a unit respawns.
+    ["CAManBase", "Killed", {
+        params ["_unit"];
+        {
+            _unit setVariable [_x, ambientTemperature select 0];
+        } forEach (_unit getVariable [QGVAR(trackedWeapons), []]);
```

Comment:
I think we could move all this to ace_overheating_fnc_handleRespawn

Reply:
Do you think it's safe to wait until respawn, rather than immediately on death?
I wouldn't want errors or weird things to happen while they are in spectator or otherwise waiting to respawn.

Reply:
@PabstMirror ?

Reply:
This is probably safer as is, so lets not change it now